main.go

```diff
@@ -110,10 +111,35 @@ sfdeploy.serverurl = https://login.salesforce.com
 sfdeploy.maxPoll = 20`
 
 func main() {
+	var projectPath string
+	flag.Parse()
+
+	if *dFlag != "" {
+		customProjectPath := *dFlag
+
+		for !strings.HasPrefix(customProjectPath, "/") || !strings.HasSuffix(customProjectPath, "/") {
+			fmt.Println("Dirctory paths must begin and end with '/'. Please enter a valid direcotry path (ex: '/Users/username/documents/'")
+			var err error
+			customProjectPath, err = bufio.NewReader(os.Stdin).ReadString('\n')
```

Comment:
Can you explain this?  Aren't you getting `customProjectPath` when you dereference `dFlag` on 118?

Reply:
So the idea is, I'm validating the user input. The way the code runs is it's expecting a '/' at the beginning and the end of the directory path. So it loops if the user doesn't enter the path correctly.

Reply:
Oh I see now.  Thanks.